Bookish/API/BooksAPIHandler.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System.Globalization;
+using System.Net;
+using Bookish.Models;
+
+namespace Bookish.API;
+using Newtonsoft.Json.Linq;
+
+public class BooksAPIHandler
+{
+   static readonly HttpClient client = new HttpClient();
+
+   private static async Task<string> MakeApiReq(string uri)
+   {
+      var responseBody = "";
+      try
+      {
+         HttpResponseMessage response = await client.GetAsync(uri);
+         response.EnsureSuccessStatusCode();
+         responseBody = await response.Content.ReadAsStringAsync();
+      }
+      catch(HttpRequestException e)
+      {
+         Console.WriteLine("\nException Caught!");	
+         Console.WriteLine("Message :{0} ",e.Message);
+      }

```

Comment:
This doesn't really handle the exception, so I wouldn't bother catching it at all